src/jdk.jpackage/windows/classes/jdk/jpackage/internal/resources/WinResources.properties

```diff
@@ -41,8 +41,9 @@ resource.overrides-wix-file=Overrides WiX project file
 resource.shortcutpromptdlg-wix-file=Shortcut prompt dialog WiX project file
 resource.installdirnotemptydlg-wix-file=Not empty install directory dialog WiX project file
 resource.launcher-as-service-wix-file=Service installer WiX project file
+resource.wix-src-conv=XSLT stylesheet converting WiX sources from WiX v3 to WiX v4 format
```

Comment:
WiX v4 -> WiX v4/v5

Reply:
The converter does exactly as described, it converts WiX v3 sources into WiX v4 **format** that can be used with WiX v4 and WiX v5. So I believe the description should remain unchanged.

Reply:
Makes sense, but for `error.no-wix-tools` I think we should add v5, in case if users sees this message when only v5 is installed.